html/pickled-tomato.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Berkshire+Swash&display=swap"
+      rel="stylesheet"
+    />
+    <title>My Asian Kitchen</title>
+
+    <link rel="stylesheet" href="../styles.css" />
+  </head>
+
+  <div style="clear: both" id="home">
+    <a class="alignleft" href="../index.html">Home</a>
+    <a class="alignright" href="appetizers.html">Appetizers</a>
```

Comment:
```suggestion
    <a class="align--left" href="../index.html">Home</a>
    <a class="align--right" href="appetizers.html">Appetizers</a>
```

I think this would be a bit better. Can follow this link for more explanation: http://getbem.com/naming/